hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneHAClusterImpl.java

```diff
@@ -661,7 +661,6 @@ private void initSCMHAConfig() {
         conf.set(scmHttpAddrKey, localhostWithFreePort());
         conf.set(scmHttpsAddrKey, localhostWithFreePort());
         conf.set(scmSecurityAddrKey, localhostWithFreePort());
-        conf.set("ozone.scm.update.service.port", "0");
```

Comment:
Is this related to this PR?

Reply:
Hi @kerneltime,

yes it is related, the CRL distribution mechanism uses the SCMUpdateService (actually the CRL distribution is the only thing uses it), and it has a config object UpdateServiceConfig which defines this config.
As in this PR I am removing the update service as it is not used by anything else, it is reasonable to remove the config option and its usages also.